src/OrbitGgp/SymbolDownloadInfoTest.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <QString>
+#include <QVector>
+
+#include "OrbitBase/Result.h"
+#include "OrbitGgp/SymbolDownloadInfo.h"
+#include "TestUtils/TestUtils.h"
+
+namespace orbit_ggp {
+
+TEST(OrbitGgpSymbolDownloadInfoTest, GetListFromJsonError) {
+  std::vector<QString> test_cases = {// Empty json
+                                     QString("{}"),
+                                     // Invalid json
+                                     QString("json"),
+                                     // One empty json object
+                                     QString(R"(
+{
+ "symbols": [{}]
+} 
+)"),
```

Comment:
You are creating QString objects here, just to convert them back into QByteArrays below. Instead you can create a vector of QByteArray here

Reply:
I didn't notice that XD. Changed accordingly. Thanks!